### ISSUE TITLE
Create fields of members without dependencies

### ISSUE THREAD

**Issue by github-nilsson** [lebab/lebab#359]
It would be great if lebab could identify and move class member properties with constant initializations to fields automatically. I.e.
```
function A(b) {
  this.a = [];
  this.b = b;
}
```
would become
```
class A {
  a = [];
  constructor(b) {
    this.b = b;
  }
}
```

**Comment by nene** [comment]
Yeah, that would be nice. But it would have to be a separate transform as it's both a stylistic preference and IIRC member fields aren't part of the older ECMAScript classes spec.

Probably best to first perform the `class` transform and then apply this one already on top of a class.

But I'm not really doing any new developments on this library... so unless somebody does a pull request, this likely won't get implemented.
